src/components/TradeWidget/TokenRow.tsx

```diff
@@ -110,16 +128,55 @@ const customSelectStyles = {
   valueContainer: (provided: CSSProperties): CSSProperties => ({ ...provided, minWidth: '4.5em' }),
 }
 
+function getMax(balance: TokenBalanceDetails | null, token: TokenDetails): string {
```

Comment:
I would use a default param for `balance` here instead of using an `||` check in the function body.

e.g `function getMax(balance: TokenBalanceDetails | null = {}, token: TokenDetails): string {`

Reply:
Hmm, `balance: TokenBalanceDetails | null = {}` results in:

```Type '{}' is missing the following properties from type 'TokenBalanceDetails': exchangeBalance, depositingBalance, withdrawingBalance, claimable, and 6 more.ts(2740)```

Couldn't yet find a way to make TS compiler happy about that.

Suggestions?

Reply:
then it's probably better not to take the object as a param and the property only

Reply:
`Pick<TokenBalanceDetails, 'which-prop-do-we-need-again'>`?